### ISSUE TITLE
JAWS 2023.2307.37 - August 2023 | Issue regarding labels not being read via aria-labelledby attributes for radio button component

### ISSUE THREAD

**Issue by Nitin-789** [FreedomScientific/standards-support#772]
### Summary

> Labels for the whole radio button group is not being read on using aria-labelledby attribute in Jaws 2023 but the same is being read in Jaws 2022 and other tools like voice over.
> 
> Example:
> 1. Go to https://jsbin.com/biredoniha/edit?html,css,js,console,output
> 2. Enable Jaws and move the focus to the radio button group

### Expected result

> Jaws should have read "Code Pen example One"

### Actual result

> Instead Jaws is only reading "One", which is incorrect this behaviour is seen in Jaws 2023 and not in Jaws 2022.

### Example

> A test case that demonstrates the issue - https://jsbin.com/biredoniha/edit?html,css,js,console,output

#### JAWS version and build number
[JAWS 2023.2307.37 - August 2023]

 
#### Operating System and version
Windows 11

#### Browser and version:
Google chrome [116.0.5845.179]

**Comment by JAWS-test** [comment]
Your example cannot work because the radio buttons are not in a labeled group. Radio buttons must be inside role=radiogroup or the fieldset element. This is not a JAWS bug

**Comment by stevefaulkner** [comment]
@Nitin-789 closing this as the example you provided is incorrectly coded. Thanks @JAWS-test for testing
